packages/eas-cli/src/build/errors.ts

```diff
@@ -0,0 +1,47 @@
+export class TurtleDeprecatedJobFormatError extends Error {
```

Comment:
General comment for this file: 
1. `Eas-build` -> `EAS Build`
2. Don't we have those error messages already defined somewhere on the server side? I think it's better to copy them over instead of rephrasing.